src/services/user.js

```diff
@@ -0,0 +1,39 @@
+
+/**
+ * User
+ *
+ * @module User
+ */
+import http from './http';
+import session from './session';
+import storage from './storage';
+
+/**
+ * Get all activities for a session, sorted alphabetically by description
+ * @param {String} sessionCode Identifier for a session
+ * @return {Promise.<User[]>}
+ */
+const getAttendedEvents = (username, termCode) =>
+  http.get(`events/chapel/${username}/${termCode}`);
+
+const getChapelCredits = async () => {
```

Comment:
This function needs a JSDoc block. Can we add one? We will want a new `@typdef` that defines what the object returned by this function looks like, too.